src/main.ts

```diff
@@ -25,6 +25,23 @@ export {
  */
 export { RobotClient as Client } from './robot';
 
+/**
+ * @deprecated Use {@link ViamClient} instead.
```

Comment:
Since this is above the export for the `ViamClient`, I don't it is accurate to mark it as deprecated and link to the ViamClient.

I believe with is a part of other TSDoc comments above protobuf exports that were previously used before dedicate classes were created in their place.